kontena-develop.yml

```diff
@@ -7,7 +7,7 @@ services:
     image: ludx/particl:0.16
     instances: 1
     volumes:
-      - /root/data/particld-dev1:/root/.particl
+      - /root/data/particld-dev1:/root/.particl:cached
     environment:
```

Comment:
this is the kontena stack file, adding cached here doesnt work and doesnt make sense

Reply:
Yeah, misplaced and didn't check.

Reply:
remove cached